stylesheets/theme5_crystallicum_dark/global.css

```diff
@@ -2206,14 +2210,21 @@ div.jqi .jqisubstate .jqibuttons button {
     text-indent: 2em;
     margin-top: 5px;
 }
+
+/* Newreply */
 .newreply {
-	color: var(--main-font-color);
+	color: var(--main-font-color); 
```

Comment:
Yes, some fields in UCP (such as writing a new PM and editing signature) are using the same class, which was also changed as I found a way to make it overall better looking (added a background to the whole table instead of adding it to each td in the table)